docs/modes-of-operation.rst

```diff
@@ -150,37 +157,37 @@ requirement, as does PostgreSQL. The BIND backend implements
 command is especially useful as it triggers an immediate retrieval of
-the zone from the configured master.
+the zone from the configured primary.
 
 Since 4.5.0, zone transfers are added to a queue and processed according to priority
 and order of addition. Order levels are (from high to low): pdns control,
 api, notify, serial changed during refresh and signatures changed during
-refresh. High priority zone transfers are always processed first, in a 
+refresh. High priority zone transfers are always processed first, in a
 first in first out order.
 
-PowerDNS supports multiple masters. For the BIND backend, the native
-BIND configuration language suffices to specify multiple masters, for
-SQL-based backends, list all master servers separated by commas in the
+PowerDNS supports multiple primaries. For the BIND backend, the native
+BIND configuration language suffices to specify multiple primaries, for
+SQL-based backends, list all primaries servers separated by commas in the
 'master' field of the domains table.
 
-Since version 4.0.0, PowerDNS requires that masters sign their
+Since version 4.0.0, PowerDNS requires that primaries sign their
 notifications. During transition and interoperation with other
 nameservers, you can use options :ref:`setting-allow-unsigned-notify` to permit
 unsigned notifications. For 4.0.0 this is turned on by default, but it
 might be turned off permanently in future releases.
 
-Master/Slave Setup Requirements
--------------------------------
+Primary/Secondary Setup Requirements
+------------------------------------
 
-Generally to enable a Master/Slave setup you have to take care of
+Generally to enable a Primary/Secondary setup you have to take care of
 the following properties.
 
-* The :ref:`setting-master`/:ref:`setting-slave` state has to be enabled in the respective ``/etc/powerdns/pdns.conf`` config files.
-* The nameservers have to be set up correctly as NS domain records i.e. defining a NS and A record for each slave.
-* Master/Slave state has to be configured on a per-domain basis in the ``domains`` table. Namely, the ``type`` column has to be either ``MASTER`` or ``SLAVE`` respectively and the slave needs a comma-separated list of master node IP addresses in the ``master`` column in the ``domains`` table. :doc:`more to this topic <backends/generic-sql>`.
+* The :ref:`setting-primary`/:ref:`setting-secondary` state has to be enabled in the respective ``/etc/powerdns/pdns.conf`` config files.
+* The nameservers have to be set up correctly as NS domain records i.e. defining a NS and A record for each secondary.
+* Primary/Secondary state has to be configured on a per-domain basis in the ``domains`` table. Namely, the ``type`` column has to be either ``PRIMARY`` or ``SECONDARY`` respectively and the secondary needs a comma-separated list of primary node IP addresses in the ``master`` column in the ``domains`` table. :doc:`more to this topic <backends/generic-sql>`.
```

Comment:
PRIMARY and SECONDARY in the type column is a bit optimistic ;) The schema and content is not yet legacy term free.